runtime/bcutil/ClassFileOracle.cpp

```diff
@@ -1344,12 +1345,27 @@ ClassFileOracle::walkMethodCodeAttributeAttributes(U_16 methodIndex)
 			StackMapFrameInfo *stackMapFramesInfo = _methodsInfo[methodIndex].stackMapFramesInfo;
 			U_8 *framePointer = stackMap->entries;
 			U_16 entryCount = U_16(stackMap->numberOfEntries);
-			for(U_16 entryIndex = 0; entryIndex < entryCount; ++entryIndex) {
-				NEXT_U8(stackMapFramesInfo[entryIndex].frameType, framePointer);
+#if defined(J9VM_OPT_VALHALLA_STRICT_FIELDS)
+			bool walkBaseFrame = false;
+#endif /* defined(J9VM_OPT_VALHALLA_STRICT_FIELDS) */
+			for(U_16 entryIndex = 0; entryIndex < entryCount;) {
```

Comment:
There is new case that we won't do `entryIndex++` after a iteration, so please consider using while loop here.